setup.py

```diff
@@ -4,17 +4,16 @@
 import subprocess
 
 from setuptools import setup, find_packages
-import setuptools.command.develop
-import setuptools.command.install
 
 cwd = os.path.dirname(os.path.abspath(__file__))
 
-version = '0.0.7'
+version = '0.0.8'
 try:
-    from datetime import date
-    today = date.today()
-    day = today.strftime("b%d%m%Y")
-    version += day
+    if not os.getenv('RELEASE'):
```

Comment:
where is RELEASE set in env?

Reply:
It is used in the CI to tag the release.